pulumi/grapl/__main__.py

```diff
@@ -182,6 +182,20 @@ def main() -> None:
         ),
     )
 
+    # To learn more about this syntax, see
+    # https://docs.rs/env_logger/0.9.0/env_logger/#enabling-logging
+    rust_log_levels = ",".join(
+        [
+            "DEBUG",
+            "h2::codec=WARN",
+            "hyper=WARN",
+            "rusoto_core=WARN",
+            "rustls=WARN",
+            "serde_xml_rs=WARN",
+        ]
+    )
+    py_log_level = "DEBUG"
+
```

Comment:
Why is this change in this PR?

Reply:
I mean, it's moving a preexisting thing around by 40 lines - but I can revert since it doesn't match the PR title

Reply:
I get that... I was just wondering why rearranging of logging code got pulled into variable renaming.